### PR TITLE
Update kubekins image to 1.24

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/666

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220624-1a63fdd9f2-1.23
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220624-1a63fdd9f2-1.24
```
Comment (Amulyam24) [inline config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits.yaml:21]: @mkumatag, one question - why are we only bumping the version of this job and not for the remaining other jobs?

Comment (mkumatag) [reply config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits.yaml:21]: good question, it was a miss, lemme add that for all the remaining jobs :)

**Comment by mkumatag** [comment]
@Amulyam24 fixed the review comment, ptal latest commit

**Review by Amulyam24 — APPROVED** [review]
/lgtm

**Pull request merged**